### PR TITLE
Redesign setup

### PR DESCRIPTION
**Pull request by Cleop**
- Sets up live-server
- Creates alternative readme with setup instructions
- Creates new index.html page
- Sets up testing on redesigned app

All from #238

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "main": "server.js",
   "scripts": {
     "test": "tape test/*.test.js",
-    "start": "node ./server.js",
+    "start": "node ./redesign/simple_server.js",
```
Comment (nelsonic) [inline package.json]: merging this PR into master will "_break_" the current website...
image

@Cleop could you _instead_ update the `dev` command to:
```
"dev": "live-server --port=8000 --open=./redesign",
```
Then use `npm run dev` on your localhost while working on the new site's `index.html`

File: redesign/simple_server.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    port: 8181,
+    host: "0.0.0.0",
+    root: "./redesign",
+    open: false,
+    ignore: 'scss,my/templates',
+    file: "./index.html",
+    wait: 1000,
+    mount: [['/components', './node_modules']],
+    logLevel: 2,
+    middleware: [function(req, res, next) { next(); }]
```
Comment (SimonLab) [inline redesign/simple_server.js]: missing closing bracket ```};``` for the params object

File: redesign/index.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>dwyl</title>
+  </head>
+  <body>
+    <h1>Hello world!</h1>
```
Comment (nelsonic) [inline redesign/index.html]: Promising. 😉

File: redesign/simple_server.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+var liveServer = require("live-server");
```
Comment (nelsonic) [inline redesign/simple_server.js]: YAGNI. 
The command described above means you can avoid having an extra file. 👍

File: package.json [hunk 5]
```diff
@@ -5,7 +5,7 @@
   "main": "server.js",
   "scripts": {
     "test": "tape test/*.test.js",
```
Comment (SimonLab) [inline package.json]: The ```test``` command needs to be updated to the future new tests. At the moment it's running the old tests of the website. Maybe we can create a test folder inside the new website with just a test which check that the "/" endpoint is loaded correctly. We can test the status code 200 for example

File: redesign_readme.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
+## How?
+
+To run this site on your localhost, simply ***install*** the *dependencies* from npm and then ***start*** it:
+
+```
+npm install && npm start
```
Comment (nelsonic) [inline redesign_readme.md]:
```
npm install && npm run dev
```

File: redesign_readme.md [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+```
+
+Then visit: http://localhost:8181
```
Comment (nelsonic) [inline redesign_readme.md]: Starting Live Server will automatically open your `default` web browser. 👍

File: package.json [hunk 8]
```diff
@@ -27,6 +27,7 @@
     "handlebars": "^4.0.10",
```
Comment (SimonLab) [inline package.json]: we can simplify the dependencies here. We are not using Hapi anymore so we could have just "live-server" as a dependency

Comment (nelsonic) [inline package.json]: @SimonLab we cannot remove the "_old_" dependencies from `package.json` until we _fully_ migrate the website else the "Production" website will fail. 👍

File: redesign/simple_server.js (hunk 2 above)
Comment (SimonLab) [inline redesign/simple_server.js]: I think we can simplify the list of parameters. Do we need to define all of them? For example ```ignore: 'scss, my/templates'" doesn't match any files in the new website. I'm not sure about the ```mount: [['/components', './node_modules']]``` property do we need it (I'll check the doc of live-server)

File: package.json (hunk 8 above)
Comment (Cleop) [inline package.json]: @SimonLab, I can't remove Hapi yet, right? so I don't break the old site ?

File: redesign/simple_server.js (hunk 2 above)
Comment (Cleop) [inline redesign/simple_server.js]: I've changed this to now look like: 
image

File: redesign/simple_server.js (hunk 4 above)
Comment (Cleop) [inline redesign/simple_server.js]: Nice and tidy 🎉

File: package.json (hunk 5 above)
Comment (Cleop) [inline package.json]: Experimented testing the server using supertest but encountered difficulty making this compatible with tape for injecting and then stopping the server. For now have set up basic tape testing without a server example.

**Pull request merged**